presto-main/src/main/resources/webapp/src/components/QueryDetail.jsx

```diff
@@ -648,6 +648,7 @@ export class QueryDetail extends React.Component {
 
```

Comment:
what is sparklines?

Reply:
line chart over time used on query details or on the main page